webui/server/server.py

```diff
@@ -16,16 +16,29 @@
 logger = logging.getLogger(__name__)
 
 # Edit this to match the serial port name shown in Arduino IDE
-SERIAL_PORT = "/dev/ttyACM0"
+SERIAL_PORT = "COM3"
 HTTP_PORT = 5000
 
 # Threads for the serial reader and writer.
 read_thread = None
 write_thread = None
 thread_stop_event = threading.Event()
 
-# L, D, U, R
-sensor_numbers = [0, 1, 2, 3]
+def init_list(length, incremental = False):
```

Comment:
We don't actually need this function.

Below do:

If you want an incremental sequence do:
```
sensor_numbers = range(num_panels)
```

If you want a list of zeroes do:
```
[0] * num_panels
```